### PR TITLE
refactor(logo.vue): migrate to vue 3 and refactor references

### PR DESCRIPTION
**Pull request by gaucj-t**
migrate components that dont extend mixins to vue 3 and refactor code base.

### CODE REVIEW

**Review by PeterThrels — APPROVED** [review]

**Review by oleggrech-t — APPROVED** [review]

**Review by PeterThrels — COMMENTED** [review]
There are several smells but beyond the scope of this task.

File: components/molecules/statements/DepositEntry.vue [hunk 1]
```diff
@@ -3,7 +3,11 @@
         <b-col cols="1">
             <div class="transaction-type-icon">
                 <div class="transaction">
-                    <img src="~/assets/svg/statement/deposit.svg" alt="" />
+                    <img
+                        alt="Deposit"
+                        loading="lazy"
+                        src="../../../assets/svg/statement/deposit.svg"
```
[the remark this answers is not in the document]
Comment (gaucj-t) [reply components/molecules/statements/DepositEntry.vue:9]: let me check this out.

File: components/atoms/TransactionAmount.vue [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+//     readonly transaction!: StatementEntryModel
+//
+//     get currency() {
+//         return weavrCurrency(this.transaction.transactionAmount)
+//     }
+//
+//     get currencyWithOp() {
+//         return weavrCurrencyWithOperator(this.transaction.sourceAmount)
+//     }
+// }
```
Comment (oleggrech-t) [inline components/atoms/TransactionAmount.vue:26]: Remove this perhaps

**Review by jasnvella2 — APPROVED** [review]

**Review by PeterThrels — APPROVED** [review]

File: components/molecules/statements/DepositEntry.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (gaucj-t) [reply components/molecules/statements/DepositEntry.vue]: should be fixed

File: components/atoms/TransactionAmount.vue (hunk 2 above)
Comment (gaucj-t) [reply components/atoms/TransactionAmount.vue]: removed

**Pull request merged**